.github/workflows/.build.yaml

```diff
@@ -34,4 +35,6 @@ jobs:
         run: |
           docker login ghcr.io -u ${GITHUB_REPOSITORY} -p ${GITHUB_TOKEN}
           docker build --tag ${{ env.image }} -f docker/Dockerfile apps/${{ github.workflow }}
+          docker tag ${{ env.image }} ${{ env.branch-image }}
```

Comment:
do we need a separate tag command?


`docker build --tag ${{ env.image }} --tag ${{ env.branch-image }} -f docker/Dockerfile apps/${{ github.workflow }}
`

Reply:
Godt poeng, det bør fungere 💡